lib/components/src/syntaxhighlighter/syntaxhighlighter.tsx

```diff
@@ -5,30 +5,30 @@ import { navigator, document, window as globalWindow } from 'global';
 import memoize from 'memoizerific';
 
 // @ts-ignore
-import jsx from 'react-syntax-highlighter/dist/cjs/languages/prism/jsx';
+import jsx from 'react-syntax-highlighter/dist/esm/languages/prism/jsx';
```

Comment:
This all LGTM, but I wonder why we were importing the CJS version before. @ndelangen @ghengeveld do you remember?

Reply:
@shilman I honestly think this might have been a bad merge maybe.

Reply:
I think what happened was that we tried it and the tests failed, as they did in this PR. The perf difference wasn't huge, so I didn't push it back then. But this time I deleted the failing test to unblock this builder work. Ideally we'd figure out how to fix the test instead. 🙈

Reply:
This is the commit that set it to `cjs` and it seems it's a problem when running Jest:

https://github.com/storybookjs/storybook/commit/2817a4972e839c924b45df228b16c5b84fad0f66

I'm using `"@storybook/components": "^6.4.22"` and having issues testing our Docs pages because of that. If I update to `6.5.0-beta.1` it works, given that version imports from `cjs`.

Here's one output from one of our tests:


```
  ● DocsPage › should render import snippet with subcomponents

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/wessouza/Sites/interface/.yarn/__virtual__/react-syntax-highlighter-virtual-f0c6b8a941/3/.yarn/berry/cache/react-syntax-highlighter-npm-13.5.3-59a76a8691-8.zip/node_modules/react-syntax-highlighter/dist/esm/languages/prism/jsx.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import jsx from "refractor/lang/jsx.js";
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (../../.yarn/berry/cache/jest-runtime-npm-28.0.2-084a36bf65-8.zip/node_modules/jest-runtime/build/index.js:1773:14)
      at Object.<anonymous> (.yarn/__virtual__/@storybook-components-virtual-398da1e650/3/.yarn/berry/cache/@storybook-components-npm-6.4.22-421ad5f3d2-8.zip/node_modules/@storybook/components/dist/cjs/syntaxhighlighter/syntaxhighlighter.js:59:35)

```